src/components/Calculator.js

```diff
@@ -1,30 +1,46 @@
 /* eslint-disable react/prefer-stateless-function */
 import { Component } from 'react';
+import calculate from './calculate';
 
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+    };
+    this.eventHandler = this.eventHandler.bind(this);
+  }
+
+  eventHandler(button) {
+    // const object = calculate(this.state, button);
+    this.setState((state) => (calculate(state, button)));
+  }
+
   render() {
+    const { total, next } = this.state;
```

Comment:
- [ ] (Optional) It would be nice to show the mathematical symbol during operations. For example, if I click `6` and click `+`, I should see `6+`. This would improve the user interaction with your calculator. 👍